versions.yaml

```diff
@@ -188,6 +188,8 @@ externals:
     # instead of abstract socket, thus kata wouldn's support the containerd's
     # version older than them.
     version: "v1.5.2"
+    # CCv0 is using a containerd PR build as our changes are still PoC
+    pr_id: "5911"
```

Comment:
Q. who is consuming this? are you going to bump it once it's merged ?

Reply:
A custom version of containerd is being used in the CCv0 CI pipeline and dev/test scripts as there are things we have added that aren't ready to merge into the main containerd project. I've been keeping the branch up to date with the container 1.5 release branch, so if/when the changes get merged in to a released container build then we'd move the CCv0 branch to use this new release. Does that help?

Reply:
thanks @stevenhorsman